code-experiments/src/coco_problem.c

```diff
@@ -53,8 +56,18 @@ void coco_evaluate_function(coco_problem_t *problem, const double *x, double *y)
 
   /* A little bit of bookkeeping */
   if (y[0] < problem->best_observed_fvalue[0]) {
-    problem->best_observed_fvalue[0] = y[0];
-    problem->best_observed_evaluation[0] = problem->evaluations;
+    is_feasible = 1;
+    if (coco_problem_get_number_of_constraints(problem) > 0) {
+      z = coco_allocate_vector(coco_problem_get_number_of_constraints(problem));
+      /* same feasibility threshold as in logger_bbob.c did not work,
+         hence using exact feasibility: */
+      is_feasible = coco_is_feasible(problem, x, z, 1.0e-5);
```

Comment:
this code should only be considered as temporary fix. The constant 1e-5 cannot stay here, as it makes a relevant difference and should not be hidden in the middle of some code.